### PR TITLE
Fix of #41

### PR DESCRIPTION
**Pull request by cyberone**
This PR fixes the issue https://github.com/yegor256/takes/issues/41 with the following:

1. Added one more ctor to PsFacebook (there are few problems because it is a facet on a facet)
1. Created PsFacebookTest class with unit-test for PsFacebook
1. Added one explicict maven dependency (was implicict)

### CODE REVIEW

**Comment by davvd** [comment]
@cyberone Thanks for the pull request, let me find a reviewer..

**Comment by davvd** [comment]
@longtimeago this pull request is for you,please review

File: src/main/java/org/takes/facets/auth/social/PsFacebook.java [hunk 1]
```diff
@@ -71,19 +84,44 @@
...
+
+    /**
+     * Ctor with proper requestor for testing purposes.
+     * @param frequest HTTP request for getting key
+     * @param frequestor Facebook response
+     * @param fapp Facebook app
+     * @param fkey Facebook key
+     * @checkstyle ParameterNumberCheck (3 lines)
+     */
+    public PsFacebook(final com.jcabi.http.Request frequest,
```
Comment (longtimeago) [inline src/main/java/org/takes/facets/auth/social/PsFacebook.java]: this constructor can be package protected if it supposed to be used for test only

**Comment by longtimeago** [comment]
@cyberone very good PR, thank you! Only one small comment above

**Comment by cyberone** [comment]
@longtimeago done, see again please.

**Comment by longtimeago** [comment]
@cyberone thank you!

**Comment by longtimeago** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@elenavolokhova please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor please deploy

**Comment by elenavolokhova** [comment]
@longtimeago 
 According to our [quality rules](http://www.teamed.io/qa.html):
> Pull request title explains the problem it is fixing.
 
 There is no meaningful title. Please ask to fix this issue and make sure that everyone follows this rule in your tickets in the future.

 > The code reviewer found at least three problems in the code.
 
 There was only one issue found during review. We should get used to this rule as soon as possible.
Please confirm that you will try to find at least three problems next time.

**Comment by longtimeago** [comment]
@elenavolokhova confirm

**Comment by elenavolokhova** [comment]
@davvd Quality is acceptable.

**Comment by elenavolokhova** [comment]
@longtimeago Thank you!

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thank you, let's try to make it "good" next time

**Comment by davvd** [comment]
@longtimeago 10 mins was added to the account of @elenavolokhova (for QA review), in transaction 56384997. **19 mins** added to your account (payment number `56385013`), many thanks for your contribution! 48 hours and 31 mins spent here.. review comments (c=4) added as a bonus. +19 to your rating, your total score is [+4765](http://www.netbout.com/b/35024?open=rating)

**Pull request merged**